c7n/resources/vpc.py

```diff
@@ -761,6 +762,12 @@ def get_eni_sgs(self):
                 sg_ids.add(g['GroupId'])
         return sg_ids
 
+    def get_codebuild_sgs(self):
+        sg_ids = set()
+        for cb in self.manager.get_resource_manager('codebuild').resources():
+            sg_ids |= set(cb['vpcConfig'].get('securityGroupIds', []))
```

Comment:
vpcconfig won't be present if it nots configured .. so cb['vpcConfig'] is a key error